MetaMask/metamask-extension
# Add Chrome store policy compliant names/descriptions

The Chrome Web Store has spam policies that prevent uploading extensions that are too similar to each other. There are exceptions for test and development versions of extensions, but these exceptions are required to be clearly identified in the name and description of the extension.

The name and description of both the Flask and beta distributions have been updated to include explicit all-caps declarations that identify them as development and beta distributions respectively, in-line with the examples shown in the Chrome Web Store spam FAQ.

For more information, see: https://developer.chrome.com/docs/webstore/spam-faq/#test-version

Manual testing steps:  
  - Run `yarn dist --build-type flask` (or download the Flask chrome build from the `metamaskbot` comment), install the build on Chrome, and ensure it has the expected title and description.
  - Run `yarn dist --build-type beta` (or download the beta chrome build from the `metamaskbot` comment), install the build on Chrome, and ensure it has the expected title and description.
